nodestream_akamai/akamai_utils/addresses.py

```diff
@@ -0,0 +1,46 @@
+"""Common utils for getting information about addresses."""
```

Comment:
This code was being reused between gtm and edns, so I pulled it up into its own util module. 

Not 100% happy with the naming, but I used up all my energy stopping myself from tweaking even more code that had nothing to do with what I wanted to fix.